arm_vm_helpers.cmake

```diff
@@ -89,6 +89,8 @@ function(DeclareCAmkESARMVM init_component)
         set(vm_plat_include "${ARM_VM_PROJECT_DIR}/components/VM_Arm/plat_include/exynos5410")
     elseif(KernelPlatformExynos5422)
         set(vm_plat_include "${ARM_VM_PROJECT_DIR}/components/VM_Arm/plat_include/exynos5422")
+    elseif(KernelPlatformZynqmpUltra96v2)
+        set(vm_plat_include "${ARM_VM_PROJECT_DIR}/components/VM_Arm/plat_include/ultra96v2")
```

Comment:
Is the zynqmp platform path deliberately missing from here?

Reply:
`zynqmp` is covered by the default `else` conditional below. Ulttra96v2 also has `KernelPlatform` be `zynqmp`, so it needs to be handled differently.